src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -521,209 +582,264 @@ record ReefPositions(Map<Pose2d, ReefScorePosition> locations, List<Pose2d> pose
+        // Tag 10 (BACK)
+        Pose2d redTag10 = new Pose2d(12.2273, 4.0259, Rotation2d.fromDegrees(180));
+        final var redTag10Robot = new Pose2d(
+                redTag10.getX() + distanceAway.in(Meters) * Math.cos(Math.toRadians(180)), 
+                redTag10.getY() + distanceAway.in(Meters) * Math.sin(Math.toRadians(180)), 
+                redTag10.getRotation()
+            );
+        scorePositionsList.add(redTag10Robot);
+        locations.put(redTag10Robot, ReefScorePosition.BACK);
+        
+        // Tag 9 (BACKRIGHT)
+        Pose2d redTag9 = new Pose2d(12.6434, 4.7455, Rotation2d.fromDegrees(120));
+        final var redTag9Robot = new Pose2d(
+                redTag9.getX() + distanceAway.in(Meters) * Math.cos(Math.toRadians(120)), 
+                redTag9.getY() + distanceAway.in(Meters) * Math.sin(Math.toRadians(120)), 
+                redTag9.getRotation()
+            );
+        scorePositionsList.add(redTag9Robot);
+        locations.put(redTag9Robot, ReefScorePosition.BACKRIGHT);
+        
+        // Tag 8 (FRONTRIGHT)
+        Pose2d redTag8 = new Pose2d(13.4744, 4.7455, Rotation2d.fromDegrees(60));
+        final var redTag8Robot = new Pose2d(
+                redTag8.getX() + distanceAway.in(Meters) * Math.cos(Math.toRadians(60)), 
+                redTag8.getY() + distanceAway.in(Meters) * Math.sin(Math.toRadians(60)), 
+                redTag8.getRotation()
+            );
+        scorePositionsList.add(redTag8Robot);
+        locations.put(redTag8Robot, ReefScorePosition.FRONTRIGHT);
+    
         scorePositions = new ReefPositions(locations, scorePositionsList);
     }
 
-    public Command runTrajectoryOdomAlign() {
-        // find closest score position to align with
-        final var pose = getPose();
-        Pose2d targetPose = pose.nearest(scorePositions.poses);
+    public Command runTrajectoryOdomAlign(AlignmentPosition position) {
         Pose2d currentPose = getPose();
-        targetXPub.set(targetPose.getX());
-        targetYPub.set(targetPose.getY());
-        targetRotPub.set(targetPose.getRotation().getDegrees());
-
-        // trajectory configuration
-        TrajectoryConfig config = new TrajectoryConfig(
-            Constants.AutoConstants.kMaxSpeedMetersPerSecond * 0.5, //TODO: tweak
-            Constants.AutoConstants.kMaxAccelerationMetersPerSecondSquared * 0.5)
-            .setKinematics(Constants.Swerve.swerveKinematics);
-        
-        // generate simple path with just start and end points
-        Trajectory trajectory = TrajectoryGenerator.generateTrajectory(
-            currentPose,
-            List.of(), 
-            targetPose,
-            config);
+        Pose2d targetPose = currentPose.nearest(scorePositions.poses);
+        ReefScorePosition reefPosition = scorePositions.locations.get(targetPose);
+        var alliance = DriverStation.getAlliance();
+        boolean isRedAlliance = alliance.isPresent() && alliance.get() == DriverStation.Alliance.Red;
```

Comment:
we should probably make a helper method for this since we seem to do it in a few places